app/assets/javascripts/admin/addon/templates/components/watched-word-form.hbs

```diff
@@ -17,6 +17,13 @@
   </div>
 {{/if}}
 
+{{#if canLink}}
+  <div class="watched-word-input">
+    <label for="watched-replacement">{{i18n "admin.watched_words.form.link_label"}}</label>
+    {{text-field id="watched-replacement" value=replacement disabled=formSubmitted class="watched-word-input-field" autocorrect="off" autocapitalize="off" placeholderKey="admin.watched_words.form.link_placeholder"}}
```

Comment:
Was wondering if we need any client side verification to ensure it is a valid HTTP URL?

Reply:
Good point, I guess we can try to parse it as a `URL` and reject anything that throws an exception/error?

Reply:
This is a good point, but checking for URLs is not an easy task. I tried using `URL` as Regis mentioned, but it did not work with relative URLs.